init/functions

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 # Location of files which contains list of protected files
-PROTECTED_FILE_LIST='/etc/qubes/protected-files.d /rw/qubes/protected-files.d'
+PROTECTED_FILE_LIST='/etc/qubes/protected-files.d'
+PER_VM_PROTECTED_FILE_LIST='/rw/qubes/protected-files.d'
```

Comment:
We have /rw/config for configuration on /rw

Reply:
OK, the patch can be altered to use that.  Done.